src/main.ts

```diff
@@ -0,0 +1,190 @@
+import chalk from "chalk";
+import fs from "fs";
+import ncp from "ncp";
+import path from "path";
+import { promisify } from "util";
+import execa from "execa";
+import Listr from "listr";
+import { spawn } from "child_process";
+import { userSupport } from "./help";
```

Comment:
It's hard for me to review your `main.ts` @tylerjusfly 
I think I now know the reason why your entire `main.ts` file changed in the github file changes in github when we were checking it the other day @tylerjusfly 
For example, check the formatting of your line 17 (& 18). In the original js file, it was only on line 17 - yours takes up lines 17 & 18.
That's just an example, there are other things reformatted in this manner.

What ever prettier settings you have either reformatted the entire file, or you just changed a lot of things without taking note.

#
Please make this easy for me to review. I prefer you redo this main.js to main.ts so that I can be sure of the changes you are making. Please do this:
- Copy the original `main.js` file from develop branch into this branch, then rename to .ts
- Add types as needed for typescript, and correct spacing of types as I have noted in previous comments.
- Ensure that things are not reformatted: e.g. maintain use of single quotes, ensure what only takes up one line doesn't take up more than one line etc.

> This way, I can see only the new changes you made and not have to review or cross check the entire file.